code/__DEFINES/time.dm

```diff
@@ -47,6 +47,12 @@ When using time2text(), please use "DDD" to find the weekday. Refrain from using
 
 #define TICKS *world.tick_lag
 
+#define MILLISECONDS * 0.10
```

Comment:
Theres 1000 milliseconds in a second, and 10 deciseconds in a second, so shouldn't there be 100 milliseconds in a second, meaning this should be *0.01?

Reply:
```suggestion
#define MILLISECONDS * 0.01
```
Yeah apparently TG goofed this up and later fixed it.